src/version/graphql.ts

```diff
@@ -0,0 +1,19 @@
+import {GitHub} from '@actions/github'
+import {GraphQlQueryResponseData} from '@octokit/graphql/dist-types/types'
+import {RequestParameters} from '@octokit/types/dist-types/RequestParameters'
+
+/**
+ * Sends a GraphQL query request based on endpoint options
+ *
+ * @param {string} token Auth token
+ * @param {string} query GraphQL query. Example: `'query { viewer { login } }'`.
+ * @param {object} parameters URL, query or body parameters, as well as `headers`, `mediaType.{format|previews}`, `request`, or `baseUrl`.
+ */
+export async function graphql(
```

Comment:
wrapper function to make it easy to mock. Otherwise hard to mock since `GitHub` is an exported class.